tests/jupyterhub_test.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+"""JupyterHub tests
+
+:copyright: Copyright (c) 2023 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+import pytest
+import os
+
+
+def setup_module(module):
+    os.environ.update(
+        SIREPO_FEATURE_CONFIG_PROPRIETARY_SIM_TYPES="jupyterhublogin",
+        SIREPO_AUTH_ROLE_MODERATION_MODERATOR_EMAIL="x@x.x",
+    )
+
+
+def test_check_auth_jupyterhub(fc):
+    fc.sr_login_as_guest()
+    # user doesn't exist
```

Comment:
This should validate that the jupyter user doesn't exist and does exist by checking the directory in srdb.

Reply:
1a08a18cb5e9c42888b493a7621bd56f9ac7faea